test/src/test/groovy/jenkins/model/PeepholePermalinkTest.groovy

```diff
@@ -71,7 +73,7 @@ class PeepholePermalinkTest {
      */
     @Test
     void legacyCompatibility() {
-        if (Functions.isWindows())  return; // can't run on windows because we rely on symlinks
+        Assume.assumeFalse("can't run on windows because we rely on symlinks", Functions.isWindows());
```

Comment:
Hmm. this is actually wrong - windows does have symlink support in jenkins.
[if it is enabled](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/Util.java#L1265) for the current user is a different issue.
This is not a `-1` on this as this is like for like replacement.